### ISSUE TITLE
训练过程中，程序卡住不动

### ISSUE THREAD

**Issue by bonn1011** [LeslieZhoa/tensorflow-facenet#15]
您好，按您最后说的那几个预处理部分存在的问题修改了程序，然后从头开始训练，但是每次训练一段时间后，程序会卡住不动，不退出也不报错。我是在Win10下运行，TensorFlow版本为1.13，Python版本为3.7。
有时候会卡在训练过程中，但更多的是卡在验证环节，print('.')的时候。后调式发现是卡在
loss_val,center_loss_val,cross_entropy_mean_val,accuracy_val,summary_val=sess.run ([total_loss,prelogits_center_loss,cross_entropy_mean, accuracy,summary_op], feed_dict=feed_dict)
这一句中。
想请教一下，这个是什么原因？

**Comment by bonn1011** [comment]
补充说明一下，怀疑是Summary导致，因此，修改代码为
loss_val,center_loss_val,cross_entropy_mean_val,accuracy_val=sess.run ([total_loss,prelogits_center_loss,cross_entropy_mean, accuracy], feed_dict=feed_dict)
后不再卡死，但为什么会这样，还没找到原因。

**Comment by ActandPersevere** [comment]
我也是卡住，原来是找个原因，我再去试试
